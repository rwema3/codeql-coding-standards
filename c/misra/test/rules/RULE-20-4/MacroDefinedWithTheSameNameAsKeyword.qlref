rules/RULE-20-4/MacroDefinedWithTheSameNameAsKeyword.ql