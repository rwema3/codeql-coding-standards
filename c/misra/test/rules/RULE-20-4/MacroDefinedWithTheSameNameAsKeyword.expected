| test.c:1:1:1:16 | #define int long | Redefinition of keyword 'int'. |
| test.c:2:1:2:30 | #define while (E) for (; (E);) | Redefinition of keyword 'while'. |
